google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/ClientSideStatementsTest.java

```diff
@@ -84,12 +98,16 @@ private static void writeLog(String statement) {
     writer.println(statement + ";");
   }
 
-  private static void openLog() {
+  private static void openLog(Dialect dialect) {
     try {
       writer =
           new PrintWriter(
               new OutputStreamWriter(
-                  new FileOutputStream(SCRIPT_FILE, false), StandardCharsets.UTF_8),
+                  new FileOutputStream(
+                      "src/test/resources/com/google/cloud/spanner/connection/"
```

Comment:
I personally find it easier to read and see where it's going by having the string literal here, instead of having to look up the constant value somewhere else.